lib/ansiblelint/rules/__init__.py

```diff
@@ -245,11 +245,37 @@ def __repr__(self) -> str:
                           for rule in sorted(self.rules, key=lambda x: x.id)])
 
     def listtags(self) -> str:
+        tag_desc = {
+            "behaviour": "Indicates a bad practice or behavior",
+            "bug": "Likely wrong usage pattern",
+            "command-shell": "Specific to use of command and shell modules",
+            "core": "Related to internal implementation of the linter",
+            "deprecations": "Indicate use of features that are removed from Ansible",
+            "experimental": "Newly introduced rules, by default triggering only warnings",
+            "formatting": "Related to code-style",
```

Comment:
I am also considering combining exiting "formatting" and "readability" tags into a "style", as I seen them to cover the same area, both aspects being part of code style. Use thump up/down for that.